src/app/interaction-model/IterationTypes.h

```diff
@@ -35,14 +35,21 @@ enum class ClusterQualityFlags : uint32_t
 
 struct ClusterInfo
 {
-    DataVersion dataVersion; // current version of this cluster
+    DataVersion dataVersion = 0; // current version of this cluster
```

Comment:
- Current cluster data version
- Why does it start at zero?

Reply:
Updated comment.

Initialization-wise I want this to do something better than "whaterver is in RAM/stack at that time" so I picked 0 as a common thing to have. It does not imply what the real versions are starting with 0. The spec says `A cluster data version SHALL be initialized randomly when it is first published` however I can hardly implement that in the constructor/initialization here :)